kloppy/domain/models/event.py

```diff
@@ -344,14 +346,34 @@ class BodyPartQualifier(EnumQualifier):
     value: BodyPart
 
 
-class GoalkeeperAction(Enum):
```

Comment:
I'm not a huge fan of replacing (renaming) this call. It means a breaking change and could break code that's currently using it. 

For me this is the only blocker to merge it.

Reply:
For consistency with "PassType", "DuelType", etc., I would say that adding "-Type" is definitely more appropriate. Maybe "GoalkeeperActionType" would be the best name here? 

Yet, I agree with Koen that this can't be changed suddenly. What about creating an alias with a deprecation warning?

Reply:
The alias approach sounds good @probberechts combined with `GoalkeeperActionType`

Reply:
@koenvo, I could use some clarification on implementing the deprecation warning? 
Does only adding a warning to the docstring suffice? Something like:
"
[DEPRECATED] GoalkeeperAction

   This class is deprecated. Use GoalkeeperActionType instead.
   "